src/ImageGallery.js

```diff
@@ -419,10 +420,10 @@ class ImageGallery extends React.Component {
       translateX = this.getTranslateXForTwoSlide(index);
     }
 
-    let translate = `translate(${translateX}%, 0)`;
+    let translate = slideVertically ? `translate(0, ${translateX}%)` : `translate(${translateX}%, 0)`;
```

Comment:
This will only update the transition but there are other considerations like swiping and the direction of chevron icons to take into account.

I think for this to work correctly two things need to happen.

```
if slideVertically = true
1. Swiping gestures updating from left/right to up/down
2. chevron left and right (< & >) updated to up and down
```